### PR TITLE
lstm trt

### PR DESCRIPTION
**Pull request by shuhand**

### CODE REVIEW

File: ODLA/platforms/tensorrt/odla_tensorrt.cc [hunk 1]
```diff
@@ -300,6 +303,16 @@ static int64_t GetTotalElements(const odla_value_shape& dims) {
 }
 
 const int nvinfer1::Dims::MAX_DIMS;
+static std::string gen_str(const nvinfer1::Dims& d) {
```
Comment (weimingzha0) [inline ODLA/platforms/tensorrt/odla_tensorrt.cc:306]: guard with some macro like 
```#ifndef NDEBUG```
?

Comment (shuhand) [reply ODLA/platforms/tensorrt/odla_tensorrt.cc:306]: ok

Comment (weimingzha0) [reply ODLA/platforms/tensorrt/odla_tensorrt.cc:306]: btw, could you squash into single commit ?

Comment (shuhand) [reply ODLA/platforms/tensorrt/odla_tensorrt.cc:306]: When merging the PR into the master, squash will be the automatic choice. Any benefit to squash commits manually?

Comment (weimingzha0) [reply ODLA/platforms/tensorrt/odla_tensorrt.cc]: I can enable "auto-merge" if it's already squashed

no significant benefits.

**Review by weimingzha0 — APPROVED** [review]

**Pull request merged**